exercises/custom-set/canonical-data.json

```diff
@@ -1,277 +1,318 @@
-            "expected": false
-         },
-         {
-            "description": "non-empty set is not equal to empty set",
-            "set1": [1, 2, 3],
-            "set2": [],
-            "expected": false
-         },
-         {
-            "description": "sets with the same elements are equal",
-            "set1": [1, 2],
-            "set2": [2, 1],
-            "expected": true
-         },
-         {
-            "description": "sets with different elements are not equal",
-            "set1": [1, 2, 3],
-            "set2": [1, 2, 4],
-            "expected": false
-         }
+        {
+          "description": "empty sets are equal",
+          "property": "equal",
+          "set1": [],
+          "set2": [],
+          "expected": true
+        },
+        {
+          "description": "empty set is not equal to non-empty set",
+          "property": "equal",
+          "set1": [],
+          "set2": [1, 2, 3],
+          "expected": false
+        },
+        {
+          "description": "non-empty set is not equal to empty set",
+          "property": "equal",
+          "set1": [1, 2, 3],
+          "set2": [],
+          "expected": false
+        },
+        {
+          "description": "sets with the same elements are equal",
+          "property": "equal",
+          "set1": [1, 2],
+          "set2": [2, 1],
+          "expected": true
+        },
+        {
+          "description": "sets with different elements are not equal",
+          "property": "equal",
+          "set1": [1, 2, 3],
+          "set2": [1, 2, 4],
+          "expected": false
+        }
       ]
-   },
-   "add": {
+    },
+    {
       "description": "Unique elements can be added to a set",
       "cases": [
-         {
-            "description": "add to empty set",
-            "set": [],
-            "element": 3,
-            "expected": [3]
-         },
-         {
-            "description": "add to non-empty set",
-            "set": [1, 2, 4],
-            "element": 3,
-            "expected": [1, 2, 3, 4]
-         },
-         {
-            "description": "adding an existing element does not change the set",
-            "set": [1, 2, 3],
-            "element": 3,
-            "expected": [1, 2, 3]
-         }
+        {
+          "description": "add to empty set",
+          "property": "add",
+          "set": [],
+          "element": 3,
+          "expected": [3]
+        },
+        {
+          "description": "add to non-empty set",
+          "property": "add",
+          "set": [1, 2, 4],
+          "element": 3,
+          "expected": [1, 2, 3, 4]
+        },
+        {
+          "description": "adding an existing element does not change the set",
+          "property": "add",
+          "set": [1, 2, 3],
+          "element": 3,
+          "expected": [1, 2, 3]
+        }
       ]
-   },
-   "intersection": {
+    },
+    {
       "description": "Intersect returns a set of all shared elements",
```

Comment:
The property is named `intersection`, should we also change the description to refer to `Intersection`?